README.md

```diff
@@ -63,6 +63,18 @@ React Starter Kit for Firebase is a popular project template (aka, boilerplate)
 
 - [react-firebase-starter](https://github.com/kriasoft/react-firebase-starter)
 
+## Preact-on-Parcel[↗](https://awesomestacks.dev/preact-on-parcel)
+
+A supersonic, featherweight, teflon-zen front-end web app starter kit. Simple beginnings, very extensible. Zero-config.
+
+- [Preact](https://preactjs.com) - [🛠](https://stackshare.io/preact) - [🐙](https://github.com/developit/preact/) - A fast 3kB alternative to React with the same modern API.
+- [Parcel](https://parceljs.org) - [🛠](https://stackshare.io/parcel) - [🐙](https://github.com/parcel-bundler/parcel) - A web application bundler, differentiated by its developer experience. It offers blazing fast performance utilizing multicore processing, and requires zero configuration.
```

Comment:
There needs to be at least 3 tools https://github.com/stackshareio/awesome-stacks/blob/master/CONTRIBUTING.md#contributing-dos-and-donts

Reply:
L'll have to get back to you on a third tool as I develop with this nanostack... Let's keep this PR on ice :D

Reply:
All good. I'm going to close it for now. Just make a new PR when you are ready =)